### PR TITLE
LoftQ: edit README.md and example files

### PR DESCRIPTION
**Pull request by yxli2123**
Hi,

I would like to update the README.md file and example scripts for LoftQ

### CODE REVIEW

File: examples/loftq_finetuning/quantize_save_load.py [hunk 1]
```diff
@@ -184,54 +183,8 @@ def quantize_and_save():
     return base_model_dir, lora_model_dir
 
 
-def load_loftq(base_model_path, lora_adapter_path):
```
Comment (younesbelkada) [inline examples/loftq_finetuning/quantize_save_load.py:187]: Why this has been removed?

**Review by younesbelkada — COMMENTED** [review]
Thank you! Left a single comment

**Review by pacman100 — APPROVED** [review]
Thank you @yxli2123 for updating README and examples for the LoftQ method, LGTM!

File: examples/loftq_finetuning/quantize_save_load.py (hunk 1 above)
Comment (pacman100) [reply examples/loftq_finetuning/quantize_save_load.py:187]: I think this function was supposed to confirm that everything works fine after the loftq weight initialization step. Hence, not a required step.

Comment (younesbelkada) [reply examples/loftq_finetuning/quantize_save_load.py:187]: Perfect!

**Pull request merged**